assemble/src/main/scripts/generate-download-script.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will generate a DEPENDENCIES listing of packaged dependencies
+
+in=target/dependencies.raw.txt
+out=target/download-dependencies
+
+cat >"$out" <<'EOF'
+#! /usr/bin/env bash
+# This script downloads the following jars, identified by their maven
+# coordinates, using the maven-dependency-plugin.
+#
+# DISCLAIMER: This is only one possible way to download a set of dependencies
```

Comment:
> But why?

Because we're not developing those projects, and there is a separation/delegation of responsibilities. We are not the upstream for those projects. We do not control the direction, compatibility, or bugfixes of those dependencies, or their release cycles. They do not fall under the Accumulo PMC responsibilities, and if we tried to take that on, I think it would be too much for our small community to cover, and it would introduce scope creep into the Accumulo project.

I think it's much healthier for us to engage and develop those connections with those other projects, and users, rather than try to take on a scope which is inherently their responsibility.

Yes, we should try to be aware of the concerns going on in our dependencies, and responsive to issues. But fixing them, making guarantees about them, or ensuring compatibility for them between a range of their versions, is outside our scope. Those dependencies communities are going to continue to move forward, after our releases. The versions of them that we use in a particular release may not be the best or recommended versions to use when a user actually downloads and uses that Accumulo release.

Do you mean relocate, like shading does? Because I'm definitely not a fan of that. That can cause a lot of hidden problems (can't relocate dynamically loaded classes, for one), and would add a lot of extra work for downstream packagers who are trying to do dependency convergence/integration under a no-bundling/dynamic-libs-only environment like most Linux distributions.